apps/comps/components/timeline-chart/constants.ts

```diff
@@ -6,9 +6,9 @@ import { createContext } from "react";
 
 export const CHART_COLORS = [
```

Comment:
if it's straightforward, we should swap these hardcoded values for the css vars you set. iirc the only place this gets used is here https://github.com/recallnet/js-recall/blob/main/apps/comps/components/timeline-chart/timeline-chart.tsx#L420

i.e., we use that array to somewhat randomly assign a chart color, while making sure it's consistent to the agent and also ensures colors aren't the same for sequential agents.